source/common/umf_pools/disjoint_pool.cpp

```diff
@@ -902,6 +904,10 @@ umf_result_t DisjointPool::initialize(umf_memory_provider_handle_t *providers,
         !((parameters.MinBucketSize & (parameters.MinBucketSize - 1)) == 0)) {
         return UMF_RESULT_ERROR_INVALID_ARGUMENT;
     }
+    // MinBucketSize parameter can't be larger than CutOff.
+    if (parameters.MinBucketSize > CutOff) {
```

Comment:
Is CutOff exposed somehow to the user? If not, how will user know what is the max size he can set?
If this is not exposed, I think you should just set MinBucketSize to CutOff if it's bigger.

Reply:
Done.